### PR TITLE
Fix typos and correct syntax for several examples.

### PR DESCRIPTION
**Pull request by kemaswill**
Fix typos and correct syntax for several examples.

### CODE REVIEW

File: example/fcn-xs/README.md [hunk 1]
```diff
@@ -5,16 +5,17 @@ This folder contains the examples of image segmentation in MXNet.
 ## Sample results
 ![fcn-xs pasval_voc result](https://github.com/dmlc/web-data/blob/master/mxnet/image/fcnxs-example-result.jpg)
 
-we have trained a simple fcn-xs model, the parameter is below:
+We have trained a simple fcn-xs model, the parameters are below:
```
Comment (piiswrong) [inline example/fcn-xs/README.md]: hyper-parameters.

File: example/fcn-xs/README.md [hunk 2]
```diff
@@ -5,16 +5,17 @@ This folder contains the examples of image segmentation in MXNet.
...
 | model   | lr (fixed) | epoch |
 | ------- | ---------: | ----: |
 | fcn-32s |      1e-10 |    31 |
 | fcn-16s |      1e-12 |    27 |
 | fcn-8s  |      1e-14 |    19 |
+
 (```when using the newest mxnet, you'd better using larger learning rate, such as 1e-4, 1e-5, 1e-6 instead, because the newest mxnet will do gradient normalization in SoftmaxOutput```)
 
-the training image number is only : 2027, and the Validation image number is: 462  
+The training dataset size is only 2027, and the validation dataset size is 462.
```
Comment (piiswrong) [inline example/fcn-xs/README.md]: Does this mean
`The training dataset has 2027 images and the validation set has 462`?

Comment (mli) [inline example/fcn-xs/README.md]: I feel it is fine.

Comment (kemaswill) [inline example/fcn-xs/README.md]: OK. Will change it if @piiswrong insists.
